### DISCUSSION TITLE
Adding Canvas element

### DISCUSSION THREAD

**Discussion by klitsner** [xemantic/shader-web-background#5] [Q&A] [answered]
Hello! I have successfully used this tool to display my shader with the default canvas that is created. However I am having trouble getting it to display on another canvas. I am trying this script which exists in the body. I have used console.log to make sure that the canvas does indeed exist when the shader is being run
```
  <script>

var myCanvas = document.getElementById("canvas");
console.log(myCanvas);

  shaderWebBackground.shade({
  canvas:{myCanvas},
      shaders: {
        image: {
          uniforms: {
            iResolution: (gl, loc, ctx) => gl.uniform2f(loc, ctx.width, ctx.height),
            iTime: (gl, loc) => gl.uniform1f(loc, performance.now() / 1000)
          }
        }
      }
    });

  
  </script>
```

The error I get reads:
_shaderWebBackground.ConfigError: config.canvas must be instance of canvas_

Is there a different way I should be referencing the canvas element? 

Thank you in advance

**Comment by morisil** [comment, marked as answer]
Have you tried replacing `canvas: {myCanvas}` with `canvas: myCanvas`?

**Reply by klitsner** [reply]
I had not... but that works! Silly error on my end, I am not sure why I thought I needed to wrap it in brackets. thank you.
